scripts/serve/serve.js

```diff
@@ -0,0 +1,16 @@
+const serve = require('webpack-serve');
+
+const webpackServe = (options) => {
+  let config = options.config;
+  if (typeof config === 'function') {
+    config = config(undefined, { p: options.production });
```

Comment:
This is assuming the webpack config functions matches toolkit's `config(env, argv)`. After reading webpack's docs on config-types, it seems this is the expected setup, however can someone provide a config only expecting env or argv? Just curious

Reply:
No, they need to match webpack api.